### PR TITLE
OAK-D Pro support, update to depthai 2.15.0.0

### PR DESCRIPTION
**Pull request by alex-luxonis**
EDIT:
Changes:
- Update `depthai` to latest release `2.15.0.0` https://github.com/luxonis/depthai-python/releases/tag/v2.15.0.0
- Support for OAK-D Pro IR dot-projector and flood-illuminator control (disabled by default, can be enabled from `Depth` tab):
image
or can be configured from command line:
```
  --irDotBrightness IRDOTBRIGHTNESS
                        For OAK-D Pro: specify IR dot projector brightness, range: 0..1200 [mA], default 0 (turned off)
  --irFloodBrightness IRFLOODBRIGHTNESS
                        For OAK-D Pro: specify IR flood illumination brightness, range: 0..1500 [mA], default 0 (turned off)
```

- StereoDepth Left-Right check enabled by default, 
the command line option `-lrc` was replaced by `-dlrc`/`--disableStereoLrCheck`

Can also be tested with the CV GUI:
`python3 depthai_demo.py -gt cv -s color left right depth`
Extra controls will appear in the `depth` window:
image

Note: the color camera (OV9782) on OAK-D-Pro-W-DEV is not supported yet.

### CODE REVIEW

File: resources/boards/OAK-D-PRO-W.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "board_config":
+    {
+        "name": "OAK-D-PRO-W",
+        "revision": "R2M1E2",
```
Comment (Luxonis-David) [inline resources/boards/OAK-D-PRO-W.json:5]:
```suggestion
        "revision": "R3M1E3",
```

File: resources/boards/OAK-D-PRO.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "board_config":
+    {
+        "name": "OAK-D-PRO",
+        "revision": "R2M1E2",
```
Comment (Luxonis-David) [inline resources/boards/OAK-D-PRO.json:5]:
```suggestion
        "revision": "R3M1E3",
```

**Review by Luxonis-David — CHANGES_REQUESTED** [review]
Small change in revision

**Comment by alex-luxonis** [comment]
Support for OAK-D Pro is available in mainline with the `2.15.0.0` release (not formally released yet, but can be installed now).
The demo was updated to use the new IR driver control API (only the CV GUI part for now).

Can be tested with:
`python3 depthai_demo.py -gt cv -s color left right depth`
`python3 depthai_demo.py -gt cv -s color left right depth -lrc`
Extra controls will appear in the `depth` window:
image

Note: the color camera (OV9782) on OAK-D-Pro-W-DEV is not supported yet, and a separate branch will still be needed (to be updated).

**Review by Erol444 — APPROVED** [review]
Tested it out with OAK-D-Pro and it works as expected :+1:

**Review by VanDavv — APPROVED** [review]
I added `--irDotBrightness` and `--irFloodBrightness` parameters and I think it's ready to be merged now @alex-luxonis

**Comment by alex-luxonis** [comment]
Updated the PR description.

**Pull request merged**